### DISCUSSION TITLE
Linking glew

### DISCUSSION THREAD

**Discussion by Im1Bear** [nigels-com/glew#312] [Q&A] [answered]
I'm trying to link GLEW to my Project but every time I want to build it in Visual Studio 2019 it says that it can't open the glew32s.lib file.

**Comment by nigels-com** [comment]
The problem is building GLEW, or the problem is linking it into another project?

**Reply by Im1Bear** [reply]
It's building GLEW

**Reply by nigels-com** [reply]
glew32s.lib is the static library target, try building that first?

**Reply by Im1Bear** [reply]
> glew32s.lib is the static library target, try building that first?
I can't and I don't know why.

**Comment by Im1Bear** [comment, marked as answer]
I now know why I got this error, even though it's a pretty stupid error.

I watched the tutorial again from front to back and made a new project, copied the glew32.dll file into debug so I don't get the LNK4098 error, and then it worked (I don't know how but it works).
